assets/source/setup-guide/app/steps/SetupTracking.js

```diff
@@ -218,9 +225,66 @@ const SetupTracking = ( { view } ) => {
 
 			setIsSaving( false );
 		},
-		[ appSettings, setIsSaving, setAppSettings, createNotice ]
+		[
+			appSettings,
+			setIsSaving,
+			setAppSettings,
+			connectAdvertiser,
+			createNotice,
+		]
 	);
 
+	const connectAdvertiser = useCallback( async () => {
+		setIsFetching( true );
+
+		try {
+			const results = await apiFetch( {
+				path:
+					wcSettings.pinterest_for_woocommerce.apiRoute +
+					'/tagowner/?advrtsr_id=' +
+					appSettings.tracking_advertiser +
+					'&tag_id=' +
+					appSettings.tracking_tag,
```

Comment:
I'd prefer to do this using template literals instead of string concatenation spanning multiple lines.

Also, with regards to https://github.com/woocommerce/pinterest-for-woocommerce/pull/305/files#r774116143, if the method is changed to `POST`, it would be good to put the advertiser and tag in POST body instead of query string.